site/content/en/docs/concepts/preemption.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Preemption"
+date: 2024-05-28
+weight: 7
+description: >
+  Preemption is the process of evicting one or more admitted Workloads to accommodate another Workload.
+---
+
+In a preemption, preempted Workloads are also called preemptees, and the ClusterQueue that
+the Workload belongs to is called a target ClusterQueue.
+Converselly, the Workload being accommodated is called the preemptor, and the ClusterQueue that the
+Workload belongs to is called a preempting ClusterQueue.
+
+## Reasons for preemption
+
+A Workload can preempt one or more Workloads if it is admitted in a [ClusterQueue with preemption enabled](/docs/concepts/cluster_queue/#preemption)
+and:
+- The preemptee belongs to the same [ClusterQueue](/docs/concepts/cluster_queue) as the preemptor and the preemptee has a lower priority.
+- The preemptee belongs to the same [cohort](/docs/concepts/cluster_queue#cohort) as the preemptor and the preemptee's ClusterQueue has a usage above
+  the [nominal quota](/docs/concepts/cluster_queue#resources) for at least one resource that the preemptee and preemptor require.
+
+The configured settings for preemption in the [Kueue Configuration](/docs/reference/kueue-config.v1beta1#FairSharing)
+and in the [ClusterQueue](/docs/concepts/cluster_queue#preemption) can limit whether a Workload can preempt others, in addition
+to the criteria above.
+
+When preempting a Workload, Kueue adds an entry in the `.status.conditions` field of the preempted Workload
+that is similar to the following:
+
+```yaml
+status:
+  conditions:
+  - lastTransitionTime: "2024-05-29T14:37:19Z"
+    message: Preempted to accommodate a higher priority Workload
+    reason: Preempted
+    status: "True"
+    type: Evicted
+```
```

Comment:
Could we also add the type=Preempted condition?

Reply:
Good idea. I rerun using kueue v0.7 and updated the status.